dev-client/src/screens/CreateProjectScreen/components/ProjectForm.tsx

```diff
@@ -149,22 +150,25 @@ export const EditForm = ({
               </Radio>
             )}
           />
-          <Fab
-            onPress={() => handleSubmit()}
-            disabled={!isValid || isSubmitting}
-            label={t('general.submit')}
-            _text={{
-              textTransform: 'uppercase',
-            }}
-            {...submitProps}
-          />
+
+          <Box position="absolute" bottom={0} right={0}>
+            <Button
+              onPress={handleSubmit}
+              isDisabled={isSubmitting || !isValid}
+              shadow={5}
+              size={'lg'}
+              display={userRole === 'MANAGER' ? 'flex' : 'none'}
+              _text={{textTransform: 'uppercase'}}>
+              {t('general.save_fab')}
+            </Button>
+          </Box>
```

Comment:
```suggestion
          {userRole === 'MANAGER'  &&
          <Box position="absolute" bottom={0} right={0}>
            <Button
              onPress={handleSubmit}
              isDisabled={isSubmitting || !isValid}
              shadow={5}
              size={'lg'}
              display={'flex'}
              _text={{textTransform: 'uppercase'}}>
              {t('general.save_fab')}
            </Button>
          </Box>
          }
```
suggestion: omit the item instead of hiding it with CSS. We can omit the button if omitting its box breaks the layout.

Reply:
👍  
Aside: I think the userRole check is maybe not even needed, because I think the entire Project Settings tab is unavailable for a non-manager to navigate to. I'm kind of hesitant to remove it since we don't have tests so I'm not confident there isn't some way though.

Reply:
That's a good point!